packages/shared/web/src/dashboard-view/hooks/use-network-tokens.tsx

```diff
@@ -26,10 +27,10 @@ export const useNetworkTokens = (
   return yieldData;
 };
 
-export const useLeveragedNTokens = (network: Network | undefined) => {
-  const model = getNetworkModel(network);
-  const yieldData = model.getTokensByType('nToken').map((t) => {
-    const debtTokens = model.getDefaultLeveragedNTokenAPYs(t);
+export const useLeveragedNTokens = () => {
```

Comment:
It seems to me like the state zero code and the code to get the data for the dashboard views are heavily duplicated. I think it would be best to consolidate the logic in both places into a single code path.